### DISCUSSION TITLE
Python - Enable automatically complete function parenthesis?

### DISCUSSION THREAD

**Discussion by iamkevindu** [zed-industries/zed#7366] [Feature Requests]
Hello, does anyone know how I could enable the parenthesis when I "tab complete" a function?

For example, I want to type 'pri' then hitting my "tab" button to autocomplete it to 'print()'. Currently, it can complete the function but it doesn't include the parenthesis.

TIA

**Comment by wxguy** [comment]
I second the proposal. This makes the difference between good and bad editor/ IDE.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
